.github/workflows/maven-deploy.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
   maven-deploy-public:
     if: ${{ github.event_name == 'workflow_dispatch' }}
-    name: Deploy to ${{ inputs.environment }}
+    name: Deploy by parameter
```

Comment:
Really? I would expect something like `oss.sonatype`

Reply:
the jobs are for automated push - it goes to jfrog all the time and the second job is for manual deployment, so you can specify the target. I change name of jobs to be more suitable..